JAVA binding compilation error

```
$ cd bindings/java
$ make                                                                                                                                                                                                                                                                         
cd .. && python const_generator.py java                                                                                                                                                                                                                                        
make -f Makefile.build all                                                                                                                                                                                                                                                     
make[1]: Entering directory '/home/pbx06/HP/TOOLS/unicorn/bindings/java'                                                                                                                                                                                                       
Warning: Nashorn engine is planned to be removed from a future JDK release                                                                                                                                                                                                     
find: ‘/usr/lib/jvm/include’: No such file or directory                                                                                                                                                                                                                        
dirname: missing operand                                                                                                                                                                                                                                                       
Try 'dirname --help' for more information.                                                                                                                                                                                                                                     
javah unicorn.Unicorn                                                                                                                                                                                                                                                          
gcc -c -fPIC -I/usr/lib/jvm/include -I -I../../include unicorn_Unicorn.c -o unicorn_Unicorn.o                                                                                                                                                                                  
In file included from unicorn_Unicorn.c:29:0:
unicorn_Unicorn.h:2:10: fatal error: jni.h: No such file or directory
 #include <jni.h>
          ^~~~~~~
compilation terminated.
Makefile.build:52: recipe for target 'unicorn_Unicorn.o' failed
make[1]: *** [unicorn_Unicorn.o] Error 1
make[1]: Leaving directory '/home/pbx06/HP/TOOLS/unicorn/bindings/java'
Makefile:4: recipe for target 'all' failed
make: *** [all] Error 2

```

--- a/bindings/java/Makefile.build
+++ b/bindings/java/Makefile.build
@@ -3,7 +3,7 @@
 
 JAVA_HOME := $(shell jrunscript -e 'java.lang.System.out.println(java.lang.System.getProperty("java.home"));')
 
-JAVA_INC := $(shell realpath $(JAVA_HOME)/../include)
+JAVA_INC := $(shell realpath $(JAVA_HOME)/include)
 
 JAVA_PLATFORM_INC := $(shell dirname `find $(JAVA_INC) -name jni_md.h`)
 

